Source/NeonAlignable.swift

```diff
@@ -125,69 +125,69 @@ public extension Alignable {
     /// match the specified `Align`. For example, if you specify `.ToTheRightMatchingTop` and provide an offset value of `5`, the
     /// view's y origin will be lower than the sibling view's y origin by 5 points.
     ///
-    public func alignAndFillWidth(align: Align, relativeTo sibling: Frameable, padding: CGFloat, height: CGFloat, offset: CGFloat = 0) {
+    public func alignAndFillWidth(_ align: Align, relativeTo sibling: Frameable, padding: CGFloat, height: CGFloat, offset: CGFloat = 0) {
```

Comment:
Most of these functions should have the required first param name as the naming convention warrants it. If the method was align() and the first param was "align", it makes sense to remove it, but given that the function name ends with "FillWidth" and the first param is the align, we should get rid of the _ here.